rotkehlchen/db/cache.py

```diff
@@ -25,5 +42,21 @@ class DBCacheDynamic(Enum):
     WITHDRAWALS_TS = 'ethwithdrawalsts_{address}'
     WITHDRAWALS_IDX = 'ethwithdrawalsidx_{address}'
 
+    @overload
+    def get_db_key(self, **kwargs: Unpack[LocationLocationNameType]) -> str:
+        ...
+
+    @overload
+    def get_db_key(self, **kwargs: Unpack[LocationLocationNameAccountIdType]) -> str:
+        ...
+
+    @overload
+    def get_db_key(self, **kwargs: Unpack[AddressType]) -> str:
```

Comment:
This is what I mean. Also why `str`? Shouldn't it be `ChecksumEvmAddress`?

```suggestion
    def get_db_key(self, **kwargs: str) -> str:
```